src/config/logtest-config.c

```diff
@@ -0,0 +1,114 @@
+/* Copyright (C) 2015-2020, Wazuh Inc.
+ * Copyright (C) 2009 Trend Micro Inc.
+ * All right reserved.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation
+ */
+
+#include "logtest-config.h"
+
+
+const char *enabled = "enabled";
+const char *threads = "threads";
+const char *max_sessions = "max_sessions";
+const char *session_timeout = "session_timeout";
+
+
+int Read_Logtest(XML_NODE node) {
+
+    for (int i = 0; node[i]; i++) {
+
+        if (!node[i]->element) {
+            merror(XML_ELEMNULL);
+            return OS_INVALID;
+        }
+
+        else if (!node[i]->content) {
+            merror(XML_VALUENULL, node[i]->element);
+            return OS_INVALID;
+        }
+
+        else if (!strcmp(node[i]->element, enabled)) {
+            if (strcmp(node[i]->content, "yes") && strcmp(node[i]->content, "no")) {
+                mwarn(XML_VALUEERR, node[i]->element, node[i]->content);
+                return OS_INVALID;
+            }
+            strcpy(w_logtest_conf.enabled, node[i]->content);
```

Comment:
I know we're sanitizing the input string, but this method can potentially cause a buffer overflow.